### PR TITLE
Add remote config size metric

### PR DESCRIPTION
**Pull request by titolins**
Fixes https://github.com/grafana/alerting-squad/issues/907

### CODE REVIEW

File: pkg/services/ngalert/metrics/remote_alertmanager.go [hunk 1]
```diff
@@ -13,72 +13,79 @@ const (
 )
 
 type RemoteAlertmanager struct {
-	Info                  *prometheus.GaugeVec
-	RequestLatency        *instrument.HistogramCollector
-	LastReadinessCheck    prometheus.Gauge
-	ConfigSyncsTotal      prometheus.Counter
+	ConfigSizeBytes       *prometheus.GaugeVec
```
Comment (titolins) [inline pkg/services/ngalert/metrics/remote_alertmanager.go:16]: Adding `ConfigSizeBytes` is the only actual change. The first commit was just alphabetically ordering the existing metrics

**Review by fayzal-g — COMMENTED** [review]
Looks good overall. Just one comment

File: pkg/services/ngalert/metrics/remote_alertmanager.go (hunk 1 above)
Comment (titolins) [reply pkg/services/ngalert/metrics/remote_alertmanager.go:16]: Ahh that makes sense. Ok, will pay attention to that

**Comment by titolins** [comment]
> Looks good overall. Just one comment

Tks a lot @fayzal-g 🙏 

It turns out this one won't be necessary though. After talking with @santihernandezc , we've decided to move those metrics to mimir instead

**Pull request closed without merging**